app/controllers/groups_controller.rb

```diff
@@ -22,11 +22,12 @@ def edit
   # POST /groups or /groups.json
   def create
     @group = Group.new(group_params)
+    @group.user_id = current_user.id
```

Comment:
You could condense this further with an association method

```suggestion
    @group = current_user.groups.new(group_params)
```

assuming a `User` has_many `groups`.